nengo/builder/ensemble.py

```diff
@@ -24,25 +24,29 @@ def sample(dist, n_samples, rng):
     return np.array(dist)
 
 
-@Builder.register(Ensemble)  # noqa: C901
-def build_ensemble(model, ens):
-    # Create random number generator
-    rng = np.random.RandomState(model.seeds[ens])
-
-    # Generate eval points
-    if isinstance(ens.eval_points, Distribution):
+def gen_eval_points(ens, eval_points, rng):
```

Comment:
Since the only thing we use from `ens` is `n_eval_points`, I'd just pass that straight in.